### PR TITLE
Add art category and APIs

### PR DESCRIPTION
**Pull request by smsalamon**
Added the "art" category and 4 relevant open APIs for art collections.

Related to #2.

### CODE REVIEW

File: src/content/apis/art/the-met.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (TheLearneer) [inline src/content/apis/art/the-met.md:1]: I checked the details and it doesn't seems to be an API. Instead a file which can be used for data purposes.

It's best if we remove this item from the list as it doesn't align with the true purpose of the repository. 🙏

File: src/content/apis/art/rijksmuseum.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: RijksData
```
Comment (TheLearneer) [inline src/content/apis/art/rijksmuseum.md:2]:
```suggestion
name: Rijks museum
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please go through the changes 🙏

File: src/content/apis/art/rijksmuseum.md (hunk 2 above)
Comment (smsalamon) [reply src/content/apis/art/rijksmuseum.md]: RijksData is the name of the site, but I changed this to the museum name. (The correct spelling is Rijksmuseum - one word.) Thanks!

**Review by TheLearneer — APPROVED** [review]
LGTM, thank you very much for your contribution @smsalamon 🙏

**Pull request merged**